lib/safira/interaction/interaction.ex

```diff
@@ -139,4 +142,57 @@ defmodule Safira.Interaction do
         {:error, changeset}
     end
   end
+
+  @doc """
+  Returns the only existing Spotlight
+  """
+  def get_spotlight() do
+    Repo.all(Spotlight) |> List.first()
+  end
```

Comment:
Probably me being dumb, is there an easier way to get the first entry in a table?